lib/peripheral.dart

```diff
@@ -28,32 +47,57 @@ class Peripheral {
           refreshGatt: refreshGatt,
           timeout: timeout);
 
+  /// Observe connection state
+  ///
+  /// Returns stream of [PeripheralConnectionState]
   Stream<PeripheralConnectionState> observeConnectionState(
           {bool emitCurrentValue = false, bool completeOnDisconnect = false}) =>
       _manager.observePeripheralConnectionState(
           identifier, emitCurrentValue, completeOnDisconnect);
 
+  /// Check connection state of the Device.
   Future<bool> isConnected() => _manager.isPeripheralConnected(identifier);
 
+  /// Disconnects from Device if it's connected or cancels pending connection.
   Future<void> disconnectOrCancelConnection() =>
       _manager.disconnectOrCancelPeripheralConnection(identifier);
 
+  /// Discovers all Services, Characteristics and Descriptors for Device.
```

Comment:
```suggestion
  /// Discovers all services, characteristics and descriptors for this peripheral.
```
Is we don't link to classes I think it's better to have this in lowercase.